.github/CODEOWNERS

```diff
@@ -99,37 +99,37 @@ common/_themeOverrides.scss @phkuo
 common/_common.scss @phkuo
 
 ## Component packages
-packages/react-avatar/ @behowell
+packages/react-avatar/ @cxe-red @behowell
 packages/react-menu/ @teams-prg
 packages/react-badge/ @ling1726 @layershifter @assuncaocharles @behowell
-packages/react-button/ @dzearing @khmakoto
-packages/react-cards/ @khmakoto
-packages/react-checkbox/ @khmakoto
+packages/react-button/ @cxe-red @dzearing @khmakoto
+packages/react-cards/ @cxe-red @khmakoto
+packages/react-checkbox/ @cxe-red @khmakoto
 packages/react-components/ @layershifter @miroslavstastny
-packages/react-focus/src/components/FocusZone/ @khmakoto
+packages/react-focus/src/components/FocusZone/ @cxe-red @khmakoto
 packages/react-image/ @cxe-prg
 packages/react-flex/ @cxe-prg
 packages/react-storybook/ @cxe-prg @ling1726 @layershifter
 packages/storybook/ @cxe-prg @ling1726 @layershifter
-packages/react-link/ @khmakoto
+packages/react-link/ @cxe-red @khmakoto
 packages/react-slider/ @microsoft/fluentui-react-pickers-and-more
```

Comment:
@behowell, do you want to move over the pickers group to be @cxe-red? It's roughly the same folks, I always intended pickers and more to be a temporary measure. Could be another PR or this one, both work.